### DISCUSSION TITLE
I found a bug in main.py

### DISCUSSION THREAD

**Discussion by Yanshengyu0311** [Astro-Sean/autophot#4] [General]
In main.py line 1014, your pipeline is:
                logging.warning('\nFWHM [%.1f]<%.1f - Sampling errors - using aperture Photometry' % (image_fwhm,autophot_input['nyquist_limit']))
I think it should be:
                logging.warning('\nFWHM [%.1f]<%.1f - Sampling errors - using aperture Photometry' % (image_fwhm,autophot_input['photometry']['nyquist_limit']))

**Comment by Astro-Sean** [comment]
Thanks! This was caught in [this issue](https://github.com/Astro-Sean/autophot/issues/7) and corrected in V1.0.4
